### PR TITLE
Add Swedish

### PR DESCRIPTION
**Pull request by magol**

### CODE REVIEW

**Comment by haacked** [comment]
Thanks for the contribution. Could you help me find another native Swedish speaker to review this PR? I unfortunately do not speak Swedish. :smile:

**Comment by magol** [comment]
hmm, it is a challenge to find someone here who speaks Swedish hear.
Is it ok if I create a issue to make it clear to Swedish speakers that we need help to review?

**Comment by haacked** [comment]
> Is it ok if I create a issue to make it clear to Swedish speakers that we need help to review?

I think this PR would serve as that issue. I'll tweet something and hope someone takes it up.

**Comment by devlead** [comment]
@Haacked @magol I can see if I can do a quick read thru.

File: lang/sv/index.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+Sammanfattning
+-------
+
+Givet ett versionsnummer MAJOR.MINOR.PATCH, räkna upp:
+
+1. MAJOR när du gör inkompatibla API-förändringar,
+1. MINOR när du lägger till funktionalitet på ett bakåtkompatibelt sätt, och
+1. PATCH när du gör bakåtkompatibla buggfixar.
+
+Ytterliga notationer finns tillgängliga som utökning till MAJOR.MINOR.PATCH-formatet
```
Comment (devlead) [inline lang/sv/index.md:18]: The word `notationer` works, but feels a bit of perhaps a word like `beteckning` is more label equivalent.

File: lang/sv/index.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+1. MINOR när du lägger till funktionalitet på ett bakåtkompatibelt sätt, och
+1. PATCH när du gör bakåtkompatibla buggfixar.
+
+Ytterliga notationer finns tillgängliga som utökning till MAJOR.MINOR.PATCH-formatet 
+för att markera testversioner och bifoga bygg-metadata.
+
+Introduktion
+------------
+
+I en värld av programhantering finns en obehaglig plats kallad “beroendeträsket”.
```
Comment (devlead) [inline lang/sv/index.md:24]: I would reword this a bit as:
`I programhanteringens värld finns det en obehaglig plats, det så kallade "beroendeträsket".`

File: lang/sv/index.md (hunk 1 above)
Comment (opinionmachine) [inline lang/sv/index.md:18]: Beteckning, va? As in, betäckning = sexy time for a mare, beteckning = designation.

Comment (devlead) [inline lang/sv/index.md:18]: Correct autocorrect😎

File: lang/sv/index.md (hunk 2 above)
Comment (opinionmachine) [inline lang/sv/index.md:24]: What is programhantering? Systemutveckling, programmering, mjukvaruutveckling I know what they are. `I systemutvecklingens värld finns en obehaglig plats, det så kallade "beroendeträsket".`

Comment (devlead) [inline lang/sv/index.md:24]: Actually using "program" as a word for "software" feels limiting.
I would rather see software translated as "mjukvara".
In that case it would read:
`"I mjukvaruutvecklingens värld finns det en obehaglig plats, det så kallade "beroendeträsket".`

Would affect many places though. What do you think @opinionmachine?

Comment (Nyarth) [inline lang/sv/index.md:24]: Mjukvaruutveckling is a better option than programhantering.

File: lang/sv/index.md [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+I en värld av programhantering finns en obehaglig plats kallad “beroendeträsket”. 
+Ju större ditt system blir och ju fler paket du integrerar i ditt program, desto 
+större är risken att du en dag finner dig själv i förtvivlans grepp. 
+
+I system med många beroenden kan nya versioner av paket snabbt bli en mardröm. Om 
+de specificerade beroendena är för hårt satta är risken stor att du hamnar i ett 
+versionslås (oförmågan att uppgradera ett paket utan att släppa nya versioner av 
+alla paket det finns beroende till). Om beroendena är specificerade för löst kommer 
+du oundvikligen hamna i en versionsnaivitet (övertro på kompabilitet med nyare 
+versioner). Du är i ett beroendeträsk om versonslås och/eller versionsnaivitet
```
Comment (Nyarth) [inline lang/sv/index.md:33]: versonslås should be versionslås.

File: lang/sv/index.md (hunk 2 above)
Comment (opinionmachine) [inline lang/sv/index.md:24]: Agreed @devlead

File: lang/sv/index.md [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+title: Semantisk versionshantering 2.0.0
+---
+
+Semantisk versionshantering 2.0.0
+==============================
+
+Sammanfattning
+-------
+
+Givet ett versionsnummer MAJOR.MINOR.PATCH, räkna upp:
```
Comment (Paxxi) [inline lang/sv/index.md:12]: I would reword this whole section as

```
Ett versionsnummer består av tre delar MAJOR.MINOR.PATCH och det finns regler för hur de ska ändras.
1. MAJOR ökas med 1 när ändringarna påverkar bakåtkompatibilitet, t ex att ett API ändras eller en metod tas bort.
1. MINOR ökas med 1 när ny funktionalitet tillkommer men 100% bakåtkompatibilitet bibehålls, t ex nya metoder läggs till.
1. PATCH ökas med 1 vid buggfixar förutsatt att 100% bakåtkompatibilitet bibehålls.

Ibland kan det även förekomma ytterligare delar i versionsnumret för att markera t ex en test eller betaversion.
```

Comment (opinionmachine) [inline lang/sv/index.md:12]: Excellent. That reads really well indeed.

Comment (magol) [inline lang/sv/index.md:12]: Very good suggestion. Making a direct translation that I made is not always the best. But is it not a bit too far from the original text? I'm not saying there is anything wrong, but still want to discuss it. If the text differs too much from the original, it becomes difficult to update when changes occur on the original text.

Comment (Paxxi) [inline lang/sv/index.md:12]: Agree that it may be more problematic to keep perfectly in sync with the English version. The upside is it works as an executive summary or tldr as the main text below is quite dense

Comment (magol) [inline lang/sv/index.md:12]: Actually, I think the original English text should be updated to match your suggestions. Your suggestions are much easier to understand than the source text. :-)

Comment (Paxxi) [inline lang/sv/index.md:12]: You're free to propose a change, I don't have the time to do it myself, just gave it a quick read when I saw a tweet from @haacked

**Comment by haacked** [comment]
Are we happy with this translation?

File: lang/sv/index.md (hunk 4 above)
Comment (magol) [inline lang/sv/index.md:12]: If we do something between the original text and you proportion like this:

```
Givet ett versionsnummer på formen MAJOR.MINOR.PATCH, räkna upp varje del enligt följande:
1. Öka MAJOR med 1 när du gör ändringar som påverkar bakåtkompatibiliteten för API:t.
1. Öka MINOR med 1 när du lägg till ny funktionalitet men 100 % bakåtkompatibilitet bibehålls.
1. Öka PATCH med 1 när du fixar buggar förutsatt att 100 % bakåtkompatibilitet bibehålls.

Ibland kan det även förekomma ytterligare delar i versionsnumret för att markera t.ex. en test- eller betaversion.
```

Comment (Paxxi) [inline lang/sv/index.md:12]: Looks good to me

Comment (opinionmachine) [inline lang/sv/index.md:12]: Yep me too

Get Outlook for iOS

On Thu, Aug 4, 2016 at 5:49 PM +0100, "Pär Björklund" notifications@github.com wrote:

> @@ -0,0 +1,249 @@
> +---
> +layout: swedish
> +title: Semantisk versionshantering 2.0.0
> +---
> +
> +Semantisk versionshantering 2.0.0
> +==============================
> +
> +Sammanfattning
> +-------
> +
> +Givet ett versionsnummer MAJOR.MINOR.PATCH, räkna upp:

Looks good to me

---

You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub:
https://github.com/mojombo/semver.org/pull/142/files/e46852c8583e765cc420cb80610764c40329c00a#r73559531

**Comment by haacked** [comment]
Tack!

**Comment by magol** [comment]
Wonderful that SemVer now is available in Swedish. I been away during the vacation, so I have unfortunately not had time to fix the things that emerged from the review. So the translation that have been merged contain spelling mistakes and not so good formulations.

Should we revert the merge so I can fix it, or should I create a new pull request with the corrections?

**Comment by haacked** [comment]
@magol Please create a new pull request. It will very much be appreciated.

**Comment by magol** [comment]
@Haacked I have now created a PR to fix the issues notated during the review process.
Sorry that I mess it up and that I have made multiple commits in the new PR, I'm a GIT noob.

I try to learn how GIT works, and I'm open for tips how to do it better next time. :-)
